test/shared/utils.py

```diff
@@ -5,6 +5,7 @@
 '''
 
 import os
+import platform as pm
```

Comment:
Is there a reason to sometimes include this as `pm` and sometimes as `platform`. Personally, I don't mind writing a few characters extra to give extra clarity and have `platform` everywhere (I did get confused for a second, as I thought it was a variable or so).

Reply:
This was done because `platform` conflicted with a string used to figure out which OS it was running on: https://github.com/joerick/cibuildwheel/pull/273/files/90da096b141d08cfb92acd086e663841423d4ac2#diff-8437219d14af4459b32737acc9521403R87

Reply:
Ah, yes, indeed. Sorry for missing that!

`platform` could be renamed to `PLATFORM`, though, now that I look at it, since it's kind of a global constant. But maybe that's not worth it.